__tests__/rtFixtureLoaderHelper.js

```diff
@@ -0,0 +1,107 @@
+// Copyright 2018 Stanford University see LICENSE for license
```

Comment:
Can this file be called `fixtureLoaderHelper.js`? The `rt` qualification is unnecessary as we don't have any other kinds of fixtures.